pkg/fakedata/completion.go

```diff
@@ -0,0 +1,70 @@
+package fakedata
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+
+	"github.com/kevingimbel/fakedata/pkg/fakedata"
+	"github.com/spf13/pflag"
+)
+
+const (
+	bashTemplate = `
+_fakedata()
+{
+    local cur prev opts
+    COMPREPLY=()
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    prev="${COMP_WORDS[COMP_CWORD-1]}"
+    opts="%s"
+
+    if [[ ${cur} == * ]] ; then
+        COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
+        return 0
+    fi
+}
+complete -F _fakedata fakedata`
+
+	zshTemplate = `
+_fakedata () {
+    local -a commands
+    IFS=$'\n'
+    commands=(%s)
+    _describe 'arguments' commands
+}
+compdef _fakedata fakedata`
+)
+
+var allCliArgs bytes.Buffer
```

Comment:
We don't. I had this inside of `init` before but then refactored it. I'll move it into `main`.